### ISSUE TITLE
the edge distance feature seems computed according to the N atom of the center residue

### ISSUE THREAD

**Issue by imSeaton** [A4Bio/ProDesign#9]
老哥，你好，还是我，再次感谢你的工作，对我帮助很大。
我注意到你的simdesign_utils.py中，第大概145行和第159的位置，计算边的方向特征时，使用的是邻居节点的所有骨干原子(N, CA, C, O)和中心节点的N原子计算的
`X = X[:,::3,:]  `这里貌似取得是中心节点N原子的全局坐标
`dX = torch.stack([X_neighbors,N_neighbors,C_neighbors,O_neighbors], dim=3) - X[:,:,None,None,:]`
请问为啥是针对中心节点的N原子与邻居中所有骨干原子计算呢，是不是用N原子计算精度更高呀？
说的不一定对，请见谅哈！
![image](https://user-images.githubusercontent.com/56856685/236463347-818ec149-4dea-4e34-841e-73a931a61ca9.png)

**Comment by gaozhangyang** [comment]
抱歉，回复迟了。这应该是个bug，我本意是想计算Ca的特征。但无论哪种，对模型性能影响应该都不大。

**Comment by Mistletoe-git** [comment]
提问的老哥你好，请问你试过把这里改成X = X[:,1::3,:]提取Ca原子坐标之后进行训练，效果变好还是变差了呢 : )
